### ISSUE TITLE
More W3C Criteria's Coverage

### ISSUE THREAD

**Issue by Ketki123-tech** [GoogleChrome/lighthouse#11966]
If we compare lighthouse with siteimprove tool in the market then siteimprove covers a lot many accessibility issues which we dont see in lighthouse as of now like below. Do we have any update by when can these be added to lighthouse too ? 

1) Invalid WAI-ARIA role, state or property - A WAI-ARIA element contains a role, state or property that is invalid or not allowed on this type of element.

2) Form control label is missing text - The form element has a label but text has not been added (label is empty).

3) Element not highlighted on focus - With keyboard navigation it is important that the focused component can be visually identified. If the focused component is not somehow highlighted, the user cannot be sure which component has focus.

**Comment by patrickhulce** [comment]
Thanks for filing @Ketki123-tech! The accessibility rules are powered by [axe](https://github.com/dequelabs/axe-core). When axe implements a rule it can then be added to Lighthouse. It might be worth asking there if they've considered these though 1 and 2 I think should both already be covered (maybe provide a specific example of case that isn't being flagged correctly?)
